### PR TITLE
Suggest number of reviewers based on change complexity

### PR DESCRIPTION
**Pull request by ikbalkaya**
As a follow up to https://github.com/ably/engineering/issues/29

I added some suggestions on which instances to seek one, two or more reviewers.

Closes https://github.com/ably/engineering/issues/29

### CODE REVIEW

**Review by AndyNicks — APPROVED** [review]
LGTM

**Review by stmoreau — APPROVED** [review]

**Comment by QuintinWillison** [comment]
@stmoreau @AndyNicks @ikbalkaya I've made some tweaks. Are they ok?

File: sdk/pull-requests.md [hunk 1]
```diff
@@ -16,6 +16,29 @@ This document is intended as a canonical reference documenting how the SDK Team
...
+Any other changes to the code will benefit from reviews from more than one person. Some example changes that are not trivial are:
+
+- Bug fixes
+- Change, Addition or deletion of functions, classes, variables, etc
+- Refactorings
+
+In some instances the number of reviewers should be increased to more than two people. Some examples of such changes are:
+
+- Public API change
+- Library struture changes - such as new modules, extensions, splits, etc.
```
Comment (ikbalkaya) [inline sdk/pull-requests.md:38]:
```suggestion
- Library structure changes - such as new modules, extensions, splits, etc.
```

**Comment by ikbalkaya** [comment]
> @stmoreau @AndyNicks @ikbalkaya I've made some tweaks. Are they ok?

@QuintinWillison I realised a typo and made a new commit but otherwise Ok from my side.

**Review by QuintinWillison — APPROVED** [review]

**Review by stmoreau — APPROVED** [review]

**Pull request merged**